packages/@aws-cdk/aws-ecs-patterns/test/fargate/integ.load-balanced-fargate-service.ts

```diff
@@ -26,7 +26,9 @@ new ApplicationLoadBalancedFargateService(stack, 'myService', {
     hostedZoneArn: 'arn:aws:route53:::hostedzone/fakeId',
     stack,
     node: stack.node,
+    account: stack.account,
```

Comment:
Feels like these shouldn't be necessary to pass in?

Reply:
Done.